Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_-_True_Positive_Incident_Handling.yml

```diff
@@ -1125,54 +1229,615 @@ tasks:
+      sender: ""
+      expired: false
+      totalanswers: 0
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "87":
+    id: "87"
+    taskid: 0cf8f731-f261-41a3-85e1-4fa4b1a9afe5
+    type: regular
+    task:
+      id: 0cf8f731-f261-41a3-85e1-4fa4b1a9afe5
+      version: -1
+      name: Tag the threat hunting results as Evidence
+      description: |
+        Adds provided entries to the incident Evidence Board. In playbook, can be positioned after a task to add the previous task's entries to Evidence Board automatically (with no need to provide arguments)
+      scriptName: AddEvidence
+      type: regular
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#none#':
+      - "88"
+    scriptarguments:
+      entryIDs:
+        simple: ${lastCompletedTaskEntries}
+      tags:
+        simple: Threat Hunting Results
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -2190,
+          "y": -490
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "88":
+    id: "88"
+    taskid: 4375e8c0-3bdd-4e5e-85a7-0cd3dd2a0f68
+    type: regular
+    task:
+      id: 4375e8c0-3bdd-4e5e-85a7-0cd3dd2a0f68
+      version: -1
+      name: Add 'Threat Hunting Results' tag to the incident
+      description: commands.local.cmd.set.incident
+      script: Builtin|||setIncident
+      type: regular
+      iscommand: true
+      brand: Builtin
+    nexttasks:
+      '#none#':
+      - "83"
+    scriptarguments:
+      appendTags:
+        simple: "true"
+      tags:
+        simple: Threat Hunting Results
+    separatecontext: false
+    continueonerrortype: ""
+    view: |-
+      {
+        "position": {
+          "x": -2190,
+          "y": -340
+        }
+      }
+    note: false
+    timertriggers: []
+    ignoreworker: false
+    skipunavailable: false
+    quietmode: 0
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "89":
+    id: "89"
+    taskid: 9050d570-9035-49f1-8b76-ebc28731cacb
+    type: condition
+    task:
+      id: 9050d570-9035-49f1-8b76-ebc28731cacb
+      version: -1
+      name: Are there any results?
+      description: Are there any results?
+      type: condition
+      iscommand: false
+      brand: ""
+    nexttasks:
+      '#default#':
+      - "83"
+      "yes":
+      - "87"
+    separatecontext: false
+    conditions:
+    - label: "yes"
+      condition:
+      - - operator: isNotEmpty
+          left:
+            value:
+              simple: Splunk
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: PANWHunting
+            iscontext: true
+        - operator: isNotEmpty
+          left:
+            value:
+              simple: Microsoft365Defender
```

Comment:
How 'Microsoft365Defender' results relevant for XDR flow?

Reply:
Part of the Threat Hunting Playbook outputs ( related more for email info)